### PR TITLE
sc-4443 system staff view skeleton (+ updated system facilities view)

### PR DESCRIPTION
**Pull request by jackofsometrade**
https://user-images.githubusercontent.com/97806603/186031847-d3f3a94c-f1bd-4bfe-b1c0-d6e1e0fe31d1.mp4

### CODE REVIEW

**Comment by jackofsometrade** [comment]
Added SystemPage as a skeleton for root level things if needed, and to provide sense to the breadcrumbs in facilities and staff pages.

File: src/pages/SystemPage.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            {
+              path: "/system",
+              breadcrumbName: `${data.facility.system.name} DOC`,
+            },
+          ],
+        }}
+        onBack={() => window.history.back()}
+      ></PageHeader>
+      <div style={WRAPPER_STYLE}>
+        <Link to={`/system/facilities`}>Facilities</Link>
```
Comment (wittejm) [inline src/pages/SystemPage.tsx:42]: I notice we don't use this `Link` anywhere in doc client but instead use antd's `Typography.Link` for in-site navigation. Let's stick with that?

File: src/graphql/GetSystemStaff.graphql [hunk 2]
```diff
@@ -0,0 +1,22 @@
+query GetSystemStaff($facilityId: ID!) {
+  facility(id: $facilityId) {
+    system {
```
Comment (wittejm) [inline src/graphql/GetSystemStaff.graphql:3]: You had mentioned the idea of adding `staff` directly to `System` in the schema/resolver. Do you still think we should do that at some point?

**Review by wittejm — APPROVED** [review]

File: src/pages/SystemPage.tsx (hunk 1 above)
Comment (jackofsometrade) [reply src/pages/SystemPage.tsx:42]: Oh ok, sure thing

File: src/graphql/GetSystemStaff.graphql (hunk 2 above)
Comment (jackofsometrade) [reply src/graphql/GetSystemStaff.graphql:3]: I do.  I was thinking we might do that if and when we need pagination?

Comment (cainlevy) [inline src/graphql/GetSystemStaff.graphql:3]: might want to ensure the query has `id` fields everywhere for proper caching

**Pull request merged**